pkg/3scale/amp/template/adapters/apicast.go

```diff
@@ -111,6 +111,9 @@ func (a *Apicast) options() (*component.ApicastOptions, error) {
 	ao.StagingPodTemplateLabels = a.stagingPodTemplateLabels()
 	ao.ProductionPodTemplateLabels = a.productionPodTemplateLabels()
 
+	// Currently, only used for monitoring resources. Thus, it does not apply to templates.
+	ao.Namespace = "mynamespace"
```

Comment:
Maybe the `${NAMESPACE}` value would work? I know the field is not being used right now in Templates but if that approach works it's probably better we set it already.

I've seen this value used on the following PR: https://github.com/openshift/origin/pull/16021/
I've never used it or checked if it exists in the templates but it seems something was done in some version related to that


`